openff/toolkit/utils/openeye_wrapper.py

```diff
@@ -6,6 +6,7 @@
 __all__ = ("OpenEyeToolkitWrapper",)
 
 
+import zstandard  # noqa
```

Comment:
zstandard isn't a hard dep of our package - I believe it's pulled in by qcportal. So this import should be wrapped in a `try/except`

```suggestion
try:
    import zstandard  # noqa
except ImportError:
    pass
```